polaris.shopify.com/src/components/PatternPage/Blocks.tsx

```diff
@@ -0,0 +1,110 @@
+import styles from './PatternPage.module.scss';
```

Comment:
Moved UsefulToKnow and HowItHelps components to a shared folder for re-use in the `MultiVariantPatternPage` and `SingleVariantPatternPage` components.